lib/jsdom/living/webstorage/Storage-impl.js

```diff
@@ -0,0 +1,98 @@
+"use strict";
+
+const DOMException = require("domexception");
+const StorageEvent = require("../generated/StorageEvent");
+const idlUtils = require("../generated/utils");
+
+// https://html.spec.whatwg.org/multipage/webstorage.html#the-storage-interface
+class StorageImpl {
+  constructor(args, { associatedWindow, storageArea, url, type }) {
+    this._associatedWindow = associatedWindow;
+    this._items = storageArea;
+    this._url = url;
+    this._type = type;
+
+    // The spec suggests a default storage quota of 5 MB
+    this._quota = 5000;
```

Comment:
@Zirro 
As I understand, 5000 would suggest that the default storage quota is 5 KB... shouldn't that be 5_000_000 bytes?